src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -90,6 +90,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     </CoreBuildDependsOn>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <CoreCleanDependsOn>
+      SuppressAssetsLogMessages;
+      $(CoreCleanDependsOn);
```

Comment:
We likely need to turn it back on after Clean because Rebuild is Clean;Rebuild

Reply:
Good catch. I'll fix that.

Reply:
I wonder if we can use $(BuildingProject) == "false" instead. It is flipped to true before CoreBuild. cc @davkean, @rainersigwald

Reply:
Probably not because that will break design-time builds from getting the diagnostics (I think).